### ISSUE TITLE
Implement `swapLeftAndRightInRTL` in 18NModule for Mac

### ISSUE THREAD

**Issue by fmsanchez** [microsoft/react-native-macos#425]
### Issue
I'm trying to position a View with 'left' layout prop
```
<View style={{left: leftPosition}}>
  {...}
</View>
```
However, my view gets positioned incorrectly whenever a RTL language is selected. This is because the default behavior is to swap left/right for RTL (context [here](https://react-mongolia.github.io/react-native/blog/page2/#facebook)). I tried calling `I18NManager. swapLeftAndRightInRTL(false)` but this doesn't seem to be implemented on Mac.

Similar issue for Windows https://github.com/microsoft/react-native-windows/issues/4662

**Comment by tom-un** [comment]
Hi @fmsanchez : I confirmed in RNTester by temporarily editing the TextExample.ios.js file that `I18nManager.swapLeftAndRightInRTL()` is working.

I had to modify the RNTester-macOS scheme as well adding arguments to force RTL mode for the app per https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/TestingYourInternationalApp/TestingYourInternationalApp.html#//apple_ref/doc/uid/10000171i-CH7-SW5

**Comment by fmsanchez** [comment]
Hey @tom-un, I was able to repro on RNTester app with the following changes:
#### **`ButtonExample.js`**
```
  {
    title: 'Simple Button',
    description: ('The title and onPress handler are required. It is ' +
      'recommended to set accessibilityLabel to help make your app usable by ' +
      'everyone.': string),
    render: function(): React.Node {
      return (
        <View style={{backgroundColor: 'blue'}}>
          <View style={{left: 10, width: 100, backgroundColor: 'red'}}>
            <Button
              onPress={() => onButtonPress('Simple')}
              testID="simple_button"
              title="Press Me"
              accessibilityLabel="See an informative alert"
            />
          </View>
        </View>
      );
    },
  }
```
#### **`RNTesterApp.ios.js`**
<pre>
<code>
type Props = {
  exampleFromAppetizeParams?: ?string,
};

<b>
I18nManager.forceRTL(true);
I18nManager.swapLeftAndRightInRTL(true);
</b>

YellowBox.ignoreWarnings([
  'Module RCTImagePickerManager requires main queue setup',
]);
</code>
</pre>

Here's a screenshot. Notice how the 'left' attribute is applied to the right of the component instead of the left. 
![Screen Shot 2020-06-24 at 5 44 30 PM](https://user-images.githubusercontent.com/4960884/85641085-6eb9f980-b642-11ea-95f5-5139d68a1f69.png)


git diff of the changes above [swapLTRRepro.txt](https://github.com/microsoft/react-native-macos/files/4828497/swapLTRRepro.txt)

**Comment by tom-un** [comment]
I made the same changes above in my clone of the fork.   I see the same issue in both macOS and iOS.   I18nManager.forceRTL() works: true makes the button hug right, false makes it hug left.   The value I18nManager.swapLeftAndRightInRTL() value seems to have no effect.

So I tried the same code in facebook master: it behaves exactly the same way.   So, as far as I can tell, the fork and upstream are working the same way.
